R/module-repository.R

```diff
@@ -411,10 +411,28 @@ setMethod(
           tmpFile <- file.path(tempdir(), "SpaDES_module_data") %>%
             checkPath(create = TRUE) %>%
             file.path(., xFile)
-          message("Downloading ", chksums$actualFile[id], " for module ", module, " ...")
-          download.file(x, destfile = tmpFile, mode = "wb", quiet = quiet)
-          copied <- file.copy(from = tmpFile, to = destfile, overwrite = TRUE)
-          destfile
+
+          if(inherits(
```

Comment:
only prompt the user if it's an interactive session. use `if(interactive())` in addition to the `inherits(..., "try-error")`.

Reply:
see also [browseURL](https://stat.ethz.ch/R-manual/R-devel/library/utils/html/browseURL.html) which may be useful for trying the url in the user's default browser.

Reply:
I don't understand how I'd use browseURL() in this case. browseURL() is making my default browser open the URL. If the URL doesn't exist I'll have a blank/error page showing in my browser.
I tried working with the value of browseURL(x), but it's always NULL, regardless of whether the URL exists or not:
x = "https://ajascuavia.het/"
str(browseURL(x))
 NULL
x = "https:://google.ca"
str(browseURL(x))
 NULL